provider/gce/export_test.go

```diff
@@ -14,10 +14,12 @@ import (
 
 var (
 	Provider                 environs.EnvironProvider = providerInstance
+	NewConfig                                         = newConfig
 	NewInstance                                       = newInstance
 	CheckInstanceType                                 = checkInstanceType
 	GetMetadata                                       = getMetadata
 	GetDisks                                          = getDisks
+	CfgBaseImagePath                                  = cfgBaseImagePath
```

Comment:
not needed?

Reply:
Good catch. This was churn from figuring out how the test suite managed config changes.